package.json

```diff
@@ -95,7 +96,8 @@
     "style-loader": "^0.23.1",
     "styled-components": "^4.2.0",
     "verdaccio": "^3.11.6",
-    "wait-on": "^3.2.0"
+    "wait-on": "^3.2.0",
+    "workerize-loader": "^1.1.0"
```

Comment:
and where is _this_ used?

Reply:
https://github.com/acl-services/paprika/pull/416  This PR is using it. (all stories, fixtures, etc)